webapp/sources/rudder/rudder-web/src/main/elm/sources/Editor/DataTypes.elm

```diff
@@ -240,6 +241,7 @@ type Msg =
   | UpdateTechnique Technique
   | DeleteTechnique (Result (Http.Detailed.Error String) ( Http.Metadata, TechniqueId ))
   | GetTechniqueResources  (Result (Http.Detailed.Error String) ( Http.Metadata, List Resource ))
+  | CopyResources  (Result (Http.Detailed.Error Bytes) ())
```

Comment:
I don't get why you have bytes here ? The resources remains in the backend in the case of cloning no ?

Reply:
because i don't care about the answer and that's how you do in Elm